Cargo.toml

```diff
@@ -14,16 +14,16 @@ panic = "abort"
 
 [workspace.dependencies]
 # workspace member deps
-sbd-bench = { version = "0.0.8-alpha", path = "rust/sbd-bench" }
-sbd-client = { version = "0.0.8-alpha", path = "rust/sbd-client" }
-sbd-e2e-crypto-client = { version = "0.0.8-alpha", path = "rust/sbd-e2e-crypto-client" }
-sbd-o-bahn-client-tester = { version = "0.0.8-alpha", path = "rust/sbd-o-bahn-client-tester" }
-sbd-o-bahn-server-tester = { version = "0.0.8-alpha", path = "rust/sbd-o-bahn-server-tester" }
-sbd-server = { version = "0.0.8-alpha", path = "rust/sbd-server" }
+sbd-bench = { version = "0.0.9-alpha2", path = "rust/sbd-bench" }
```

Comment:
Moving to `alpha2` since this is a breaking change and there's a chance we'll need to apply a fix to the version used in hc 0.4.